modules/oke/locals.tf

```diff
@@ -18,7 +18,7 @@ locals {
   policy_statement = (var.use_encryption == true) ? "Allow dynamic-group ${oci_identity_dynamic_group.oke_kms_cluster[0].name} to use keys in compartment id ${var.compartment_id} where target.key.id = '${var.kms_key_id}'" : ""
 
   # policy to allow block volumes inside oke to use kms
-  oke_volume_kms_policy_statements = [
+  oke_volume_kms_policy_statements = (var.node_pool_volume_kms_key_id == "") ? []: [
```

Comment:
You are submitting a PR for operator but also changing a functionality in OKE. In future PRs, please change only that for which you are submitting a PR.

Reply:
sure, will keep that in mind